tests/test_global.sh

```diff
@@ -2,11 +2,14 @@
 
 # Output checking
 ./irpc.py ./examples/simple.irp.c > tests/simple.c
-diff ./tests/gold.simple.c tests/simple.c
+diff -w -b -B ./tests/gold.simple.c tests/simple.c
+
+./irpc.py ./examples/touch.irp.c > tests/touch.c
+diff -w -b -B ./tests/gold.touch.c tests/touch.c
```

Comment:
Please add a test for the compilation.
You can split test_global in `test_simple` and `test_touch`. Maybe more readable